packages-ts/starknet-gauntlet/src/wallet/defaultWallet.ts

```diff
@@ -0,0 +1,25 @@
+import { ec, KeyPair, Signer } from 'starknet'
+import { IStarknetWallet } from './'
+import { Env } from '../dependencies'
+
+export class Wallet implements IStarknetWallet {
+  wallet: Signer
```

Comment:
Wallet has a wallet?
```suggestion
  signer: Signer
```